types/muibox/index.d.ts

```diff
@@ -13,14 +15,22 @@ export interface Dialog {
     prompt(options: PromptOptions | string): Promise<string>;
 }
 
+export interface DialogButtonOptions {
+    text: string;
```

Comment:
According to the docs, there's a default:
```suggestion
    text?: string;
```